libraries/Microsoft.Bot.Builder.LanguageGeneration/Range.cs

```diff
@@ -38,6 +53,10 @@ public Range(int startLine, int startChar, int endLine, int endChar)
         /// </value>
         public Position End { get; set; }
 
+        /// <summary>
+        /// Returns a string that represents the current object.
+        /// </summary>
+        /// <returns>A string value.</returns>
         public override string ToString()
```

Comment:
I'm not sure we really need documentation on these overrides.

Reply:
I changed to `/// <inheritdoc/>`.